client/src/components/ErrorPage/Error.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter, MemoryRouter, Route } from 'react-router-dom';
+import Button, { ButtonStyles } from 'components/Button/Button';
+import Routes from 'components/Routes';
+import Main from 'pages/Main';
+import SecondaryPagesLayout from 'components/layouts/SecondaryPagesLayout';
+import Error from './Error';
+
+const testIds = {
+  title404: 'title-404',
+  button: 'button',
+  layout: 'layout'
+};
+
+describe('Error tests', () => {
+  test('error is rendering', async () => {
+    <MemoryRouter>{render(<Error />, { wrapper: BrowserRouter })}</MemoryRouter>;
+    expect(screen.getByText(/404 - No page found/i)).toBeInTheDocument();
+    expect(screen.getByTestId(testIds.title404)).toBeInTheDocument();
+  });
+
+  test('button is rendering', async () => {
+    render(<Button type="button" children="Return Home" className={ButtonStyles.ErrorBtn} />);
+    expect(screen.getByTestId(testIds.button)).toBeInTheDocument();
+  });
+
+  test('SecondaryPagesLayout is rendering', async () => {
+    render(<SecondaryPagesLayout children={undefined} />);
+    expect(screen.getByTestId(testIds.layout)).toBeInTheDocument();
+  });
```

Comment:
in file 'Error.spec.tsx' you should add tests for Error component only, tests for Button, SecondaryPagesLayout and Main should be placed in there own files